pkg/ebpf/tracer.go

```diff
@@ -711,6 +711,11 @@ func SectionsFromConfig(c *Config) map[string]bpflib.SectionParams {
 		tcpCloseEventMap.sectionName(): {
 			MapMaxEntries: 1024,
 		},
-		"socket/dns_filter": {},
 	}
+
+	if isRHELOrCentos, _ := isRHELOrCentOS(); isRHELOrCentos {
```

Comment:
Should we also take into account whether or not DNS collecting is enabled?

Reply:
👍 will update

Reply:
+1 here